frontend/src/pages/UserInformations/UserInformations.scss

```diff
@@ -0,0 +1,125 @@
+@import "../../variable.scss";
+
+.viewport_userinformations::before {
+  content: " ";
+  position: absolute;
+  top: 0;
+  width: 100vw;
+  height: 25vh;
+  background: $gradient90deg;
+  z-index: -1;
+  border-bottom: $borderDark;
+}
+
+.viewport_userinformations {
+  height: 100vh;
+  display: grid;
+  grid-template-columns: 4rem 1fr 2fr 8%;
+  grid-template-rows: 20% 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 3fr 1fr;
+  justify-items: center;
+  align-items: center;
+  font-size: 1.1rem;
+}
+
+.userpicture {
+  height: 30vh;
+  border-radius: 41px 100% 100% 100%;
+  margin: 1rem 0 0 1rem;
+  box-shadow: $boxShadow_inv;
+  border: 1px solid $colorPrimary;
+  grid-area: 1 / 1 / 4 / 4;
+  justify-self: start;
+  align-self: start;
+}
+
+.viewport_userinformations .icon {
+  width: 2.2rem;
```

Comment:
here too

Reply:
rem intented here, so it follows text increase when zoomed